debian/patches/series

```diff
@@ -1 +1 @@
-rm-py2-modules
+rm-python2
```

Comment:
More Python 2-only code has come into the package, which is a problem for `pycompile`, a command that is injected by the `dhpython` command. I thought about trying to shortcut this behavior, but since its standard, I'll leave it as it is.. instead I've patched our source code to delete Python 2-only code via a [Quilt](https://wiki.debian.org/UsingQuilt) patch.